### DISCUSSION TITLE
is it viable that analyzing protocol by MITM attack?

### DISCUSSION THREAD

**Discussion by zhaodice** [Melledy/Grasscutter#1092] [Ideas]
establish fake server that client connect to ,but the fake server connect to the real server, is it viable analysis way by man in the middle attack

**Comment by AZthemute** [comment]
So, using a proxy?

**Reply by zhaodice** [reply]
> So, using a proxy?

likes a proxy but not all,it maybe a chain:

official client-> fake server -> [record network conversation data] -> fake client -> official server

or we can inject game to get decrypt key or non-encrypted data.

**Comment by Birdulon** [comment]
AFAIK that's how a lot of research has been done so far on the protocol, using Iridium I think.
